chart/flux/README.md

```diff
@@ -273,6 +274,7 @@ The following tables lists the configurable parameters of the Flux chart and the
 | `syncGarbageCollection.dry`                       | `false`                                              | If enabled, fluxd won't delete any resources, but log the garbage collection output (see [garbage collection](/docs/references/garbagecollection.md))
 | `manifestGeneration`                              | `false`                                              | If enabled, fluxd will look for `.flux.yaml` and run Kustomize or other manifest generators
 | `hostAliases`                                     | `{}`                                                 | Additional hostAliases to add to the Flux pod(s). See <https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/>
+| `priorityClassName`                               | `""`                                                 | The name of the priority class to assign to the Flux pod(s).
```

Comment:
```suggestion
| `priorityClassName`                               | `""`                                                 | The name of the priority class to assign to the Flux pod.
```

As the replica count is a hardcoded `1` and the release strategy `Replace`.

Reply:
This entry can be removed from the table as it was already added in another PR.

Reply:
Done!